src/models/User/createUser/integration.spec.js

```diff
@@ -89,21 +81,28 @@ describe('The User model’s `createUser` helper', () => {
 
   it('should add a user to the database and return it', async done => {
     const countBefore = await database('users').count()
-    const { user } = await createUser(validUser)
+    const { user } = await createUser({
+      name: 'Foo',
+      email: 'foo@bar.com',
+      passwordPlain: '$uper$trongPa$$word'
+    })
     const countAfter = await database('users').count()
 
-    expect(countBefore[0].count).toEqual('0')
-    expect(countAfter[0].count).toEqual('1')
+    expect(countBefore[0].count).toEqual('1')
+    expect(countAfter[0].count).toEqual('2')
     expect(user).toMatchObject({
-      id: 1,
```

Comment:
Let’s re-add the `id` since we run in band.
I wonder, why do we not use `.toEqual` instead of `toMatchObject` here too? I forgot why I did that.